samcli/commands/check/resources/Graph.py

```diff
@@ -0,0 +1,19 @@
+"""
```

Comment:
General comment for Graph/LambdaFunction/TemplateResource classes:
Are these better fitted for `lib` folder instead `commands` folder?

Reply:
I kept them in the command (check/Resources) folder because they are designed for the check command. I'm not sure if they can/will be used for stuff other than `sam check`, given that the parameters seem to be unique to the `check` functionality.

Reply:
I don't have strong feelings for this. These classes seem to be purely model and don't relate to CLI.
We can keep them here and, if needed in the future, moving them should be relatively easy.